### ISSUE TITLE
Support for paths with and without the forward-slash

### ISSUE THREAD

**Issue by burtlo** [hashicorp/secrets-store-csi-driver-provider-vault#34]
The use of the forward-slash before the object path caused me some confusion. It feels different from a lot of the other places. I would like for the provider to support with a forward-slash and without a forward-slash.

```
...
    objects:  |
      array:
        - |
          objectPath: "/foo"                    # secret path in the Vault Key-Value store e.g. vault kv put secret/foo bar=hello
          objectName: "bar"
          objectVersion: ""
```

```
...
    objects:  |
      array:
        - |
          objectPath: "foo"                    # secret path in the Vault Key-Value store e.g. vault kv put secret/foo bar=hello
          objectName: "bar"
          objectVersion: ""
```

**Comment by burtlo** [comment]
I lost some time because I thought the forward-slash being specified meant that the `objectPath` that it wanted the full path to include the `data` for kv-v2. Which created issues as it intelligently added an additional data to it.

**Comment by tomhjp** [comment]
Addressed in #63 - it now supports resolving all of the following to the same path (and doesn't add `/data/` as per #35):
- `secret/foo`
- `/secret/foo`
- `v1/secret/foo`
- `/v1/secret/foo`
